docs/_deployments/serverless-framework.md

```diff
@@ -0,0 +1,342 @@
+> 💡 Customize the [region][aws-cli-region] and [output format][aws-cli-output-format] best for you.
+
+That wraps up configuring your local machine to access AWS. 👏 Next, let's do the same with the Serverless Framework.
+
+---
+
+### Set up Serverless Framework
+
+Now we can set up the free [Serverless Framework](https://www.serverless.com/open-source/) open source tools on your local machine. These tools allow you to easily configure, debug, and deploy your app to AWS Lambda.
+
+**1. Install the Serverless Framework CLI**
+
+The Serverless tools are available as a Command Line Interface (CLI) and can be [installed on macOS, Windows, or Linux](https://www.serverless.com/framework/docs/getting-started/). You can install it using npm with the command:
+
+```shell
+npm install --save-dev serverless
+```
+
+> 💡 Alternatively, you can [globally install the Serverless CLI](https://www.serverless.com/framework/docs/getting-started/) using the command `npm install -g serverless`.
+
+Once the installation is complete, you can test the Serverless CLI by displaying the commands available to you:
+
+```shell
+npx serverless help
+```
+
+You're now set up with the Serverless tools! Let's move on to preparing your Bolt app to run as an AWS Lambda function.
+
+---
+
+### Get a Bolt Slack app
+
+If you haven't already built your own Bolt app, you can use our [Getting Started guide][getting-started-guide] or clone the template app below:
+
+```shell
+git clone https://github.com/slackapi/bolt-js-getting-started-app.git
+```
+
+After you have a Bolt app, navigate to its directory:
+
+```shell
+cd bolt-js-getting-started-app/
+```
+
+Now that you have an app, let's prepare it for AWS Lambda and the Serverless Framework.
+
+---
+
+### Prepare the app
+
+**1. Prepare the app for AWS Lambda**
+
+In this section, we'll update your Bolt app to respond to a Lambda function instead of listening for HTTP requests. We'll do this by creating a custom Bolt receiver for Lambda.
+
+First, install the official [AWS Serverless Express](https://github.com/awslabs/aws-serverless-express) module to transform Express HTTP requests to Lambda function events:
+
+```bash
+npm install aws-serverless-express
+```
+
+Next, update the [source code that imports your modules](https://github.com/slackapi/bolt-js-getting-started-app/blob/main/app.js#L1) in `app.js` to require Bolt's Express receiver and the AWS Serverless Express module:
+
+```javascript
+const { App, ExpressReceiver } = require('@slack/bolt');
+const awsServerlessExpress = require('aws-serverless-express');
+```
+
+Then update the [source code that initializes your Bolt app](https://github.com/slackapi/bolt-js-getting-started-app/blob/main/app.js#L3-L7) to create a custom receiver using AWS Serverless Express:
+
+```javascript
+// Initialize your custom receiver and ack Slack requests after handled
+const expressReceiver = new ExpressReceiver({
+  signingSecret: process.env.SLACK_SIGNING_SECRET,
+  processBeforeResponse: true
```

Comment:
We may want to mention this a bit more clearly. This option is required for FaaS environments while it's not for other situations. Also, it is the only reason why we manually instantiate the receiver here.

Reply:
This is very good feedback, thanks @seratch. I'll take expand the comment to explain the importance and loop back with you!

Reply:
@seratch I've now added a detailed comment explaining the purpose of `processBeforeResponse`. What are your thoughts? Is it too length? Is the comment block above the option the ideal place? 

https://github.com/slackapi/bolt-js/pull/698/files#diff-05c75277f909db812ad95893db7558f993e8e51ff8f596273609b70ee2fa2ff9R132-R140

Reply:
@mwbrooks Looks great 👍